<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWin</class>
 <widget class="QWidget" name="LoginWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登陆界面</string>
  </property>
  <widget class="QLineEdit" name="user_password">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user_login">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_number">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>账户</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>基于Reid的全景智能安防系统——边缘端</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
